hydra-node/src/Hydra/Chain.hs

```diff
@@ -129,6 +129,8 @@ data PostTxError tx
     -- NOTE: PlutusDebugInfo does not have much available instances so we put it
     -- in Text form but it's lame
     PlutusValidationFailed {plutusFailure :: Text, plutusDebugInfo :: Text}
+  | -- | User tried to commit more than 100 ADA hardcoded limit on mainnet
+    ReachedMainnetHardcodedLimit
```

Comment:
Must: Add what the hard-coded limit is *and* what the user tried to commit into the error response.

This should be simple and makes it much more convenient to react accordingly on the client side.